src/Catel.MVVM/Catel.MVVM.Shared/Services/NavigationService.xamarin.forms.cs

```diff
@@ -98,56 +127,52 @@ public override void RemoveAllBackEntries()
             throw new MustBeImplementedException();
         }
 
+        /// <summary>
+        /// Initialize
```

Comment:
This comment is useless, please remove or create a valid doc comment. Since this one is partial (thus private), it can safely be removed since we don't have to document private members.